File: src/main/java/com/p2p/lending/service/DetailsService.java
package com.p2p.lending.service;

import java.util.List;

import com.p2p.lending.pojo.Details;

public interface DetailsService {

	public List<Details> detailslist(Integer pid);

	int delete(Integer id);
	void deleteByPid(Integer id);
	int create(Details details);

	Details get(Integer id);

	int update(Details record);

	 
}

File: src/main/java/com/p2p/lending/service/DetailsService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Details

interface DetailsService {

    fun detailslist(pid: Int?): List<Details>

    fun delete(id: Int?): Int
    fun deleteByPid(id: Int?)
    fun create(details: Details): Int

    operator fun get(id: Int?): Details

    fun update(record: Details): Int


}
